plugins/default/lib/openshift.py

```diff
@@ -72,7 +72,8 @@ def _get_running_pod_containers(oc, project, selector=None):
     for p in pods:
         if p["status"]["phase"] == "Running":
             for c in p["spec"]["containers"]:
-                result.append((p["metadata"]["name"], c["name"], c))
+                if (not container_names or c["name"] in container_names):
+                    result.append((p["metadata"]["name"], c["name"]))
```

Comment:
Need to add back returning the container object (c), the memory and cpu checks use it see https://github.com/feedhenry/nagios-container/blob/master/plugins/default/lib/memory_usage.py#L101:

```
result.append((p["metadata"]["name"], c["name"], c))
```